### PR TITLE
Use common_library

### PR DESCRIPTION
**Pull request by eile**

### CODE REVIEW

**Comment by eile** [comment]
Don't merge - will need to sync with other projects

**Comment by eile** [comment]
Retest this please

File: eq/admin/CMakeLists.txt [hunk 1]
```diff
@@ -1,62 +1,46 @@
...
+  pipe.cpp
+  segment.cpp
+  server.cpp
+  view.cpp
+  window.cpp
+  )
+
+set(EQUALIZERADMIN_LINK_LIBRARIES EqualizerFabric EqualizerServer)
+set(PROJECT_INCLUDE_NAME eq/admin)
+set(EQUALIZERSERVER_OMIT_PROJECT_HEADER ON)
```
Comment (tribal-tec) [inline eq/admin/CMakeLists.txt]: EQUALIZERADMIN_OMIT?

File: eq/node.h [hunk 2]
```diff
@@ -19,9 +19,9 @@
 #ifndef EQ_NODE_H
 #define EQ_NODE_H
 
-#include <eq/client/api.h>
-#include <eq/client/types.h>
-#include <eq/client/visitorResult.h>  // enum
+#include <eq/api.h>
+#include <eq/types.h>
+#include <eq/visitorResult.h>  // enum
 #include <eq/fabric/node.h>           // base class
```
Comment (tribal-tec) [inline eq/node.h]: comment indent

File: eq/server/CMakeLists.txt [hunk 3]
```diff
@@ -15,18 +15,15 @@ list(APPEND CPPCHECK_EXTRA_ARGS
...
 endif()
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-
-add_subdirectory(equalizers)
-
 add_definitions(-DYY_NEVER_INTERACTIVE)
 
-set(HEADERS
+set(PUBLIC_HEADERS
```
Comment (tribal-tec) [inline eq/server/CMakeLists.txt]: EQUALIZERSERVER_PUBLIC_HEADER?

File: eq/util/texture.h [hunk 4]
```diff
@@ -20,7 +20,7 @@
 #ifndef EQUTIL_TEXTURE_H
 #define EQUTIL_TEXTURE_H
 
-#include <eq/client/frame.h>        // Frame::Buffer enum
+#include <eq/frame.h>        // Frame::Buffer enum
 
 #include <lunchbox/thread.h>         // thread debug macro
```
Comment (tribal-tec) [inline eq/util/texture.h]: comment indent

File: eq/worker.cpp [hunk 5]
```diff
@@ -1,21 +1,21 @@
...
- * 
+ *
  * You should have received a copy of the GNU Lesser General Public License
  * along with this library; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#include "api.h"
+#include <eq/api.h>
 #define CO_WORKER_API EQ_API
```
Comment (tribal-tec) [inline eq/worker.cpp]: this still holds true? Looks legacy

File: eq/admin/CMakeLists.txt (hunk 1 above)
Comment (eile) [inline eq/admin/CMakeLists.txt]: good catch

File: eq/worker.cpp (hunk 5 above)
Comment (eile) [inline eq/worker.cpp]: Might be. Hard to test without a Windows box.

**Comment by eile** [comment]
All done except CO_WORKER_API

**Comment by tribal-tec** [comment]
Nah, I think the CO_WORKER_API seems still needed. Exported function in a templated class from another library. Ancient developers probably had reasons to make it work like this...

**Comment by tribal-tec** [comment]
+1

File: eq/CMakeLists.txt [hunk 6]
```diff
@@ -1,13 +1,293 @@
...
   include_directories(SYSTEM ${CUDA_INCLUDE_DIRS})
 endif()
 
+install(FILES DESTINATION include/eq COMPONENT dev)
+
+list(APPEND CPPCHECK_EXTRA_ARGS -DEQUALIZER_DSO_NAME=foo -DEQ_API=)
+
+add_definitions(-DEQ_PLUGIN_BUILD -DBOOST_PROGRAM_OPTIONS_DYN_LINK)
+
+set(EQUALIZER_PUBLIC_HEADERS
```
Comment (bilgili) [inline eq/CMakeLists.txt]: Putting all those into a Files.cmake ?

Comment (eile) [inline eq/CMakeLists.txt]: That's how it was before, but then the CMakeLists is almost empty.

**Pull request merged**